postgres/README.md

```diff
@@ -0,0 +1,191 @@
+# TDK PostgreSQL Demo
+
+
+[Pagila](https://github.com/devrimgunduz/pagila) is a sample database schema that can be used in MySQL tutorials, articles, demos, etc.
+
+```shell
+git clone https://github.com/synthesized-io/pagila-tdk-demo
+cd pagila-tdk-demo/postgres
+```
+
+Prepare an [inventory file](https://docs.synthesized.io/tdk/latest/user_guide/reference/inventory) - [inventory.yaml](inventory.yaml).
+
+
+## Generation of the empty database from scratch
```

Comment:
I suggest renaming the title to:

## Generation based on the empty schema

Reply:
Fixed in [30b6f4e](https://github.com/synthesized-io/pagila-tdk-demo/pull/12/commits/30b6f4eb179902e9925d50ec80550b431a1012ca)